pandas/io/formats/style.py

```diff
@@ -913,7 +941,10 @@ def set_td_classes(self, classes: DataFrame) -> Styler:
         ...     ["min-val red", "", "blue"],
         ...     ["red", None, "blue max-val"]
         ... ], index=df.index, columns=df.columns)
-        >>> df.style.set_td_classes(classes)
+        >>> df.style.set_td_classes(classes)  # doctest: +SKIP
+            A	B   C
+        0   1   2   3
+        1   4   5   6
```

Comment:
same comment as previous.. output not necessary as a distractor: `set_td_classes` affects the class object and the HTML result.